internal/graph/check.go

```diff
@@ -591,7 +651,10 @@ func (c *LocalChecker) checkTTU(parentctx context.Context, req *ResolveCheckRequ
 			tuple.NewTupleKey(object, tuplesetRelation, ""),
 		)
 		if err != nil {
-			return &openfgapb.CheckResponse{Allowed: false}, err
+			return &ResolveCheckResponse{
+				Allowed:            false,
+				ResolutionMetadata: &ResolutionMetadata{},
```

Comment:
`var emptyResolutionMetdata = &ResolutionMetadata{}` and then use `emptyResolutionMetadata` here and elsewhere.